src/views/AddItem.jsx

```diff
@@ -4,11 +4,36 @@ import React, { useState } from 'react';
 //LOCAL IMPORTS
 import { addItem } from '../api/firebase.js';
 
-export function AddItem({ listToken }) {
+export function AddItem({ listToken, data }) {
 	// SET STATES
 	const [itemName, setItemName] = useState('');
 	const [days, setDays] = useState(7);
 	const [status, setStatus] = useState(null);
+	// NON RegEx VERSION FOR CLEANING INPUT
+	// const cleanedUpItems = (userInput) =>{
+	// 	const alphabet = 'abcdefghijklmnopqrstuvwxyz'
+	// 	const acceptedCharacters = alphabet+alphabet.toUpperCase()+'0123456789'
+	// 	let acceptableString = ''
+	// 	for (let i=0; i<userInput.length; i++){
+	// 		const goodChar = userInput[i]
+	// 		if(acceptedCharacters.includes(goodChar)){
+	// 			acceptableString += goodChar
+	// 		}
+	// 	}
+	// 	return acceptableString.toLowerCase()
+	// }
+
+	// const trimmedItem = cleanedUpItems(itemName)
+	// const names = data.map((item) => cleanedUpItems(item.name))
+
+	// RegEX VERSION, SAME RESULT AS ABOVE
+	const iLoveRegex = /[^a-z0-9]/g;
+	const cleanedUpItems = (userInput) => {
```

Comment:
I suggest using more clear variable names so that it's easier to understand the purpose of each. For `iLoveRegex`, could use something like nonAlphaNumeric. For `cleanedUpItems`, would be better to use the present tense of the verb "cleaned" and since you're only cleaning up one item at a time, use the singular "item" ==> `cleanUpItem`.

Reply:
+1 to Jenny's comment